trulens_eval/examples/expositional/end2end_apps/custom_app/custom_agent.py

```diff
@@ -0,0 +1,92 @@
+from typing import Any, Optional
```

Comment:
Can you move all the stuff from this PR in `expositional` out like we discussed?

Reply:
Moved and added to the examples README.md the description of `experimental` and the new `dev` folder where the dummy app stuff has been moved to.